domain/src/main/java/org/oppia/android/domain/profile/ProfileManagementController.kt

```diff
@@ -887,7 +888,7 @@ class ProfileManagementController @Inject constructor(
           ProfileAvatar.newBuilder().setAvatarImageUri(imageUri).build()
       } else {
         updatedProfile.avatar =
-          ProfileAvatar.newBuilder().setAvatarColorRgb(colorRgb).build()
+          colorRgb.let { color -> ProfileAvatar.newBuilder().setAvatarColorRgb(color).build() }
```

Comment:
In an earlier commit, I had set `colorRgb` to be nullabe, so this was meant for nullsafety. But I have since refactored the code so that `colorRgb` is not nullable, and I have also reverted this change.